src/Core/Content/Cms/DataResolver/CmsSlotsDataResolver.php

```diff
@@ -177,6 +177,8 @@ private function fetchByCriteria(array $searches, SalesChannelContext $context):
         $searchResults = [];
         foreach ($searches as $definitionClass => $criteriaObjects) {
             foreach ($criteriaObjects as $criteriaHash => $criteria) {
+                $criteria->addState(Criteria::STATE_ELASTICSEARCH_AWARE);
```

Comment:
@vienthuong is this safe to always add or should we check for the definition as well? 🤔